data/channels.csv

```diff
@@ -2914,10 +2926,10 @@ Canal22Internacional.mx,Canal 22 Internacional,,,,MX,,,c/US,spa,,FALSE,,,,https:
 Canal22MetropolitanayNacional.mx,Canal 22 Metropolitana y Nacional,,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://cdn.mitvstatic.com/channels/mx_canal-22-conaculta_m.png
 Canal22Nacional.mx,Canal 22 Nacional,,,,MX,,,c/MX,spa,,FALSE,,,,https://www.canal22.org.mx/,https://imagenes.gatotv.com/logos/canales/oscuros/22_de_mexico-mediano.png
 Canal23Mazatenango.gt,Canal 23 Mazatenango,,,,GT,GT-10,,s/GT-07;s/GT-09;s/GT-10;s/GT-11,spa,,FALSE,,,,https://www.playgt.net/canal23,https://i.imgur.com/ToCBqRk.png
-Canal24Horas.es,Canal 24 Horas,,RTVE,,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
-Canal24HorasCanarias.es,Canal 24 Horas Canarias,,RTVE,,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
-Canal24HorasCatalunya.es,Canal 24 Horas Catalunya,,RTVE,,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
-Canal24HorasInternacional.es,Canal 24 Horas Internacional,,RTVE,,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24Horas.es,Canal 24 Horas,,24,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasCanarias.es,Canal 24 Horas Canarias,,24,RTVE,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasCatalunya.es,Canal 24 Horas Catalunya,,24,RTVE,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasInternacional.es,Canal 24 Horas Internacional,,24,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png

```

Comment:
I assume this is a typo, because the 4th column should be the name of the network.

Reply:
OK. I delete the name for now. I move the name as owner only.

Reply:
Wait a minute, would it be okay to set the name of the network as "Canal 24"?

Reply:
The value of "network" is still incorrect, according to the description on wikipedia it is "Televisión Española" (https://en.wikipedia.org/wiki/24_Horas_(Spanish_TV_channel))